src/technologies.js

```diff
@@ -109,6 +111,7 @@ const technologies = [
   { name: "Python 3", released: new Date("2008-12-03"), icon: "python", link: "https://www.python.org/" },
   { name: "PyTorch", released: new Date("2016-08-26"), link: "https://pytorch.org/" },
   { name: "R", released: new Date("1993-08-01"), links: "https://www.r-project.org/" },
+  { name: "RabbitMQ", released: new Date("2007-02-08"), links: "https://www.rabbitmq.com/"},
```

Comment:
change the property name to `link` (singular)

Reply:
Done.